tests/package/Makefile

```diff
@@ -12,6 +12,14 @@ clean:
 %/mesos-version: target/mesos-version
 	cp target/mesos-version $@
 
+# 1.5.0-2.0.1 package has issues for Debian, and 1.5.0-2.0.2 isn't released for all distros. When we move past this,
+# remove these build steps
+debian-systemv/mesos-version: target/mesos-version
```

Comment:
Is there a ticket somewhere to remove those? I fear it will be forgotten...

Reply:
I feared the same thing. It's a little bit hard to read (had to put it on one line because of makefile's), but, the build will fail if this workaround is still present and the Mesos version is no longer `1.5.0-2.0.1`.
```
if [ $$(cat target/mesos-version) = "1.5.0-2.0.1" ]; then
  echo "1.5.0-2.0.2" > $@
else
  echo "Package version patch should not be needed; remove this step from the Makefile"
  exit 1
fi
```

Reply:
Oh, right! Or we could insert s `sleep` and make each subsequent test run slower until someone notices :D

Reply:
I have to admit that I don't quite get these steps. Why are we echoing `1.5.0-2.0.2` when the version is `1.5.0-2.0.1` even though `1.5.0-2.0.2` is not released?

Reply:
What do you think about writing `1.5.0.-2.0.2` in `Dependencies.scala` for `MesosDebian` and fall back to `1.5.0-2.0.1` if `1.5.0-2.0.2` does not exist.

Reply:
1.5.0-2.0.2 is only released for the Debian and the Ubuntu 14 Linux distributions. It is not available for Centos. Historically, the same package version has been released for all distributions of Linux.

I have a preference to keep it as is; it solves the immediate use case and has a built-in mechanism to make sure we remove this workaround in the future when it is no longer needed

Reply:
Your comment made me realize that we do not have a package dependency for Marathon RPM packages. This is something that we should probably fix. Click add single comment